CMakeLists.txt

```diff
@@ -185,6 +185,14 @@ if(ADDRESS_SANITIZER)
 endif()
 
 #------------------------------------------------------------------------------
+# Enable fingerid_test
+#------------------------------------------------------------------------------
+option(ENABLE_FINGERID_TEST "Enable test for fingerid (SiriusAdapter)" OFF)
+if(ENABLE_FINGERID_TEST)
+   set(enable_fingerid = true)
```

Comment:
I don't think this does what you want. I think it sets the variable to a list of "=" and "true" (which probably evaluates to true anyway if you check the variable in a boolean way).
See https://cmake.org/cmake/help/v3.0/command/set.html
But maybe this is not needed. See comment above.